bot/interact.go

```diff
@@ -81,6 +81,10 @@ func interactAPIHandler(botContext cmd.BotContext) http.HandlerFunc {
 					if minSalary > maxSalary {
 						validationErrors["min_salary"] = "The Salary Min field should contain a lower value than the specified in Salary Max field."
 					}
+
+					if (minSalary * 2) > maxSalary {
+						validationErrors["max_salary"] = "The Salary Max field should be at most twice the value specified in Salary Min field."
```

Comment:
Creo que en vez de un mensaje de error con la regla de negocio, un mensaje de error con una explicación puede ser más valioso:

```suggestion
			 			validationErrors["max_salary"] = "The gap between MinSalary and MaxSalary is rather large. Maybe you should post two different job offers with different responsibilities and required qualifications. Salary is a relevant field, we recommend you try to keep it meaningful to increase the chances of taking the position seriously by potential candidates."
```